src/commands/synthetics/README.md

```diff
@@ -431,6 +431,15 @@ The name of the custom subdomain set to access your Datadog application. If the
 
 Pass a query to select which Synthetic tests to run.
 
+The syntax for this query is the same as the one used in the [Synthetics tests list page's search bar](https://app.datadoghq.com/synthetics/tests).
```

Comment:
```suggestion
The syntax for this query is the same as that used in the [Synthetics tests list page's search bar][14].
```

Reply:
In addition to the above, could you please add the `[14]` reference link on line `1042` as below?
`[14]: https://app.datadoghq.com/synthetics/tests`

I'm unable to make a suggestion on that part of the file.

Reply:
@buraizu thanks for the review 🙇